src/org/labkey/remoteapi/internal/EncodeUtils.java

```diff
@@ -0,0 +1,41 @@
+package org.labkey.remoteapi.internal;
+
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+public class EncodeUtils
```

Comment:
Not sure if we have unit tests in this package but this class' utilities would be good candidates for coverage.

Reply:
Sadly, there are no tests in this package. There is the `JavaClientApiTest` Selenium test, but no unit tests.